wkcuber/test_api.py

```diff
@@ -0,0 +1,194 @@
+from PIL import Image, ImageFilter
+
+from wkcuber.api.Dataset import WKDataset, TiffDataset
+from wkcuber.api.Slice import TiffSlice
+import numpy as np
+from skimage import io
+
+if __name__ == "__main__":
+
+    ds = WKDataset.create("/home/robert/Desktop/temp/simple_wk_dataset", [1])
+    mag = ds.add_layer("color", "color", num_channels=3).add_mag(
```

Comment:
Something that I noticed during the demo earlier:  I found it hard to read/understand`ds.add_layer("color", "color"`. One `color` refers to a category and the other to a name, right? Perhaps the categories can be constants/enums. e.g. `Dataset.ColorType` or `DS.ColorLayer` or `DS.COLOR` or something.

`mag = ds.add_layer(Dataset.ColorType, "color", num_channels=3).add_mag(...`